packaging/amazon-linux-ami/ecs-init.spec

```diff
@@ -27,11 +27,11 @@ Source1:        ecs.conf
 
 BuildRequires:  golang >= 1.7
 
-Requires:       docker >= 1.6.0, docker <= 17.03.2ce
+Requires:       docker = 17.03.2ce
```

Comment:
We'll want to update both our SUSE and Ubuntu packages as well to have the new version.

@rjschwei Does SUSE have a new-enough version of Docker?  I think that 1.13 is the earliest we'd be able to work with, as long as `--init` functions properly.

Reply:
Sure, but i'm keeping that as out of scope for this pr

Reply:
Yes, current version of docker on SLES is 1.8.2 and 1.10.2 is in the pipeline.

Reply:
@rjschwei Sounds like SLES does *not* have a new-enough version of Docker.  We may want to start with `--pid=host` on SLES instead of `--init`.

Reply:
Ah, sorry I didn't read that correctly, but with SLES 12 SP3, to be released on Thursday:

~> rpm -qa docker
docker-17.04.0_ce-98.2.x86_64

So going forward we are OK, given that the SP2 images will be replaced by SP3, although SP3 will still have version 1.13.0 I don't think we need anything special :smile: